src/_pytest/compat.py

```diff
@@ -354,3 +354,10 @@ def funcargnames(self):
 
     def overload(f):  # noqa: F811
         return f
+
+
+attrs_has_eq = getattr(attr, "__version_info__", (0, 0)) >= (19, 2)
```

Comment:
can probably just be `getattr(..., ())` -- are there versions of attrs which don't provide `__version_info__`? can we grab this attribute unconditionally?

Reply:
19.2.0 introduced `__version_info__` specifically for the eq/order case, so sadly it has to be this way.

Reply:
oof, ok -- can still do the `()` thing if you want, though it's not a big deal